src/PrestaShopBundle/Resources/views/Admin/Module/Includes/action_menu.html.twig

```diff
@@ -23,42 +23,63 @@
+    <div class="btn-group form-action-button-container">
+      <form class="btn-group form-action-button" method="post" action="{{ urls[url_active] }}">
+        <button type="submit" class="btn btn-primary-reverse btn-primary-outline light-button module_action_menu_{{ url_active }}"
+          data-confirm_modal="module-modal-confirm-{{ name }}-{{ url_active }}"
+          {% if
+            level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_UPDATE') or
+            (
+              level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE') and
+              url_active not in ['enable', 'disable', 'configure', 'install', 'upgrade']
+            )
+          %}
+            disabled="disabled"
+          {% endif %}>
           {{ url_active|capitalize|replace({'_': " "})|trans({}, 'Admin.Actions') }}
-      </button>
-      {% if urls|length > 1 %}
-      <input type="hidden" class="btn">
-      {% endif %}
-    </form>
-    {% if (urls|length > 1) %}
-      {% if (level > constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_CREATE')) or ((('configure' in urls|keys) and ('upgrade' in urls|keys)) and  url_active in ['configure', 'install', 'upgrade']) %}
+        </button>
+        {% if urls|length > 1 %}
+          <input type="hidden" class="btn">
+        {% endif %}
+      </form>
+      {% if (urls|length > 1) %}
+        {% if
+          level > constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_CREATE') or
+          (
+            (('configure' in urls|keys) and (('upgrade' in urls|keys) or ('enable' in urls|keys) or ('disable' in urls|keys))) and
+            url_active in ['enable', 'disable', 'configure', 'install', 'upgrade']
+          )
+        %}
           <button type="button" class="btn btn-primary-outline  dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
             <span class="caret"></span>
             <span class="sr-only">{{ 'Toggle Dropdown'|trans({}, 'Admin.Modules.Feature') }}</span>
           </button>
           <div class="dropdown-menu">
             {% for module_action, module_url in urls %}
               {% if module_action != url_active %}
-                {% if ((level >= constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE')) or ((level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE')) and (module_action in ['configure', 'install', 'upgrade']))) %}
+                {% if
+                  level >= constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE') or
+                  (
+                    level < constant('PrestaShopBundle\\Security\\Voter\\PageVoter::LEVEL_DELETE') and
+                    module_action in ['enable', 'disable', 'configure', 'install', 'upgrade']
```

Comment:
Here is the list of possible actions:

* 'install'
* 'uninstall'
* 'enable'
* 'disable'
* 'enable_mobile'
* 'disable_mobile'
* 'reset'
* 'upgrade'

I think we still forgot `enable_mobile` and `disable_mobile`. But it would be better to put instead 
```php
module_action not in ['uninstall', 'reset']
```

ping @kompilorb for confirmation.